app/uk/gov/nationalarchives/omega/editorial/views/editSetRecordEdit.scala.html

```diff
@@ -443,7 +444,7 @@ <h3 class="govuk-heading-s">@messages("edit-set.record.edit.separated-material")
         @govukErrorSummary(ErrorSummary().withFormErrorsAsText(editSetRecordForm))
     }
     <h1 class="govuk-heading-l">@messages("edit-set.record.edit.heading", record.ccr)</h1>
-    <h2 class="govuk-heading-m">@messages("edit-set.record.edit.id", record.oci)</h2>
+    <h2 class="govuk-heading-m">@messages("edit-set.record.edit.id", record.oci) @recordType</h2>
```

Comment:
Couldn't we send the record type as the second arg to the message?

Reply:
Done

Reply:
Almost there - you just need to resolve that into a message.

As it is, it comes out as `PhysicalRecord` instead of `Physical Record`.

Reply:
Confirmed.